specification/communications-manager.yaml

```diff
@@ -33,20 +33,7 @@ paths:
     get:
       $ref: endpoints/get_message.yaml
   /channels/nhsapp/accounts:
-    parameters:
```

Comment:
These are both duplicated under the `get:` endpoint below so aren't required here